doit/cmd_info.py

```diff
@@ -23,6 +23,16 @@ def my_safe_repr(obj, context, maxlevels, level):
     return pprint._safe_repr(obj, context, maxlevels, level)
 
 
+opt_show_build_reason = {
+    'name': 'show_build_reason',
+    'short': 'R',
+    'long': 'show-build-reason',
```

Comment:
I chose `R` for `reason` (I didn't want to use lowecase `r` because I tend to associate that with `recursive`). But `status` sounds good, too, with shortcut `s`.